config/config.exs

```diff
@@ -47,6 +47,20 @@ config :logger, :console,
 # Use Jason for JSON parsing in Phoenix
 config :phoenix, :json_library, Jason
 
+config :lunchbot, Lunchbot.Scheduler,
+  jobs: [
+    # Bldr times
+    {"0 18 * * mon",
+     {Lunchbot.Slackbot, :sendLunchReminder, ["bldr", "https://lunchbot.withmojo.com"]}},
+    {"0 18 * * thu",
+     {Lunchbot.Slackbot, :sendLunchReminder, ["bldr", "https://lunchbot.withmojo.com"]}},
+    # PVD times
+    {"0 16 * * mon",
```

Comment:
I wonder if you could combine both rows as `0 16 * * 2,5` to run on monday and thursday